pbs/pbsrequest_test.go

```diff
@@ -450,3 +450,91 @@ func TestParseMobileRequest(t *testing.T) {
 		t.Errorf("Parse sdk platform failed")
 	}
 }
+
+func TestParseMalformedMobileRequest(t *testing.T) {
+	body := []byte(`{
```

Comment:
What's the difference between this test and the earlier test `TestParseMobileRequestFirstVersion`? They look the same to me

Reply:
No SDK version is passed in this request. The sample request client sent did not have SDK version, that's a separate issue to investigate but this test case is to cover that.